structuredefinitions/Extension-CareConnect-VaccinationProcedure-1.xml

```diff
@@ -28,195 +28,6 @@
   <type value="Extension" />
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
   <derivation value="constraint" />
-  <snapshot>
```

Comment:
I'm guessing the snapshot has been removed due to Forge. Given the other extensions still have snapshot, we possibly want to ensure this is still present here.

Reply:
Looked at the UKCore page (https://simplifier.net/guide/UK-Core-Implementation-Guide-STU3-Sequence/Home/ProfilesandExtensions/ExtensionLibrary/Extension-UKCore-VaccinationProcedure.page.md?version=current) , <snapshot> is not there, so didn't include that